web/shapeworks/src/views/Main.vue

```diff
@@ -80,11 +80,10 @@ export default defineComponent({
         onMounted(async () => {
             try {
                 await loadDataset(props.dataset);
-                await loadProjectForDataset(props.project, props.dataset);
+                await loadProjectsForDataset(props.dataset);
```

Comment:
I think we may have been wrong here to change this line. When we first mount the main page, we need to load in `selectedDataset` and `selectedProject` from the url parameters. If we fail to do that, we need to take the user back to the select page. We changed this to get all the projects available for a dataset. Could you please change this line back (if the store function still exists, use that)?

Reply:
This is supposed to load the projects and select the given project?

Reply:
Added `loadProjectInDataset` in ca4efa0